Library/Formula/jet.rb

```diff
@@ -0,0 +1,14 @@
+class Jet < Formula
+  desc "CLI for Codeships Docker infrastructure"
+  homepage "https://codeship.com/documentation/docker/release-notes/"
+  url "https://s3.amazonaws.com/codeship-jet-releases/0.13.0/jet-darwin_amd64_0.13.0.tar.gz"
```

Comment:
Bottles are built by our CI bot. If there’s no source release it should go to our [`homebrew/binary`](https://github.com/Homebrew/homebrew-binary) tap.